robyn/__init__.py

```diff
@@ -302,110 +310,193 @@ def inner(handler):
 
         return inner
 
-    def get(self, endpoint: str, const: bool = False, auth_required: bool = False):
+    def get(
+        self,
+        endpoint: str,
+        const: bool = False,
+        auth_required: bool = False,
+        openapi_tags: List[str] = ["get"],
+    ):
         """
         The @app.get decorator to add a route with the GET method
 
-        :param endpoint str: endpoint to server the route
+        :param endpoint str: endpoint for the route added
+        :param const bool: represents if the handler is a const function or not
+        :param auth_required bool: represents if the route needs authentication or not
+        :param openapi_tags: List[str] -- for grouping of endpoints in the openapi spec
```

Comment:
```suggestion
        :param openapi_tags: list[str] -- for grouping of endpoints in the openapi spec
```

This should be the types for the entire file